package/MDAnalysis/analysis/helix_analysis.py

```diff
@@ -339,75 +345,48 @@ class HELANAL(AnalysisBase):
         'local_screw_angles': (-2,),
     }
 
-    @classmethod
-    def from_SecondaryStructure(cls, secondary_structure, select='name CA',
```

Comment:
The SecondaryStructure PR is nowhere near done, this can get added back in that one.